modules/fundamental/Cargo.toml

```diff
@@ -17,9 +17,12 @@ actix-http = { workspace = true }
 actix-web = { workspace = true }
 anyhow = { workspace = true }
 async-graphql = { workspace = true, features = ["uuid", "time"] }
+async-trait = { workspace = true }
```

Comment:
I _think_ that latest Rust version we're using means we don't need `async-trait` crate any more.

Reply:
When I remove it and it's use I get this compiler error: 

```
error[E0195]: lifetime parameters or bounds on method `call` do not match the trait declaration
  --> modules/fundamental/src/ai/service/tools.rs:26:18
   |
26 |     async fn call(&self, input: &str) -> Result<String, Box<dyn Error>> {
   |                  ^ lifetimes do not match method in trait
```

Reply:
I'll gander Monday.